core/common/src/test/java/alluxio/PropertyKeyTest.java

```diff
@@ -270,15 +270,15 @@ public void isBuiltIn() {
 
   @Test
   public void impersonationRegex() {
-    String name = "user-name";
-
     // test groups
+    String name = "a-a.b-b.c-c.groups";
```

Comment:
Nit: include upper case, underscore and digits in the test case

Reply:
added more characters.